crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -161,6 +160,24 @@ where
 }
 
 impl ConductorConfig {
+    /// The most minimal config, which will not work
+    pub fn empty() -> Self {
+        Self {
+            tracing_override: None,
+            data_root_path: None,
+            keystore: KeystoreConfig::default(),
+            dpki: DpkiConfig::testing(),
+            admin_interfaces: None,
+            network: KitsuneP2pConfig::empty(),
+            db_sync_strategy: DbSyncStrategy::default(),
+            tuning_params: None,
+            device_seed_lair_tag: None,
+            danger_generate_throwaway_device_seed: false,
+            #[cfg(feature = "chc")]
+            chc_url: None,
+        }
+    }
```

Comment:
I removed the Default impl, in place of this, to highlight the fact that the default config will not work, due to there being no sensible default tx5 config (to my knowledge)